build.gradle.kts

```diff
@@ -25,7 +25,7 @@ plugins {
     jacoco
 }
 group = "com.ampnet"
-version = "0.3.1"
+version = "0.3.2"
```

Comment:
Current version in master branch is `0.3.2`, did you pull the latest changes before pushing the branch? I suggest you do this locally:
```bash
git checkout master
git pull
git checkout -
git rebase master # hopefully you will have no conflicts, if you do then simply resolve them
git push --force
```

After that set the version to `0.3.3`. Or maybe just set the version, whichever you want is fine.